msg/vehicle_status.msg

```diff
@@ -71,10 +72,13 @@ bool rc_signal_lost				# true if RC reception lost
 uint8 rc_input_mode				# set to 1 to disable the RC input, 2 to enable manual control to RC in mapping.
 
 bool data_link_lost				# datalink to GCS lost
-bool high_latency_data_link_active 		# all low latency datalinks to GCS lost
 uint8 data_link_lost_counter			# counts unique data link lost events
+
+bool high_latency_data_link_lost 			# all low latency datalinks to GCS lost
```

Comment:
I think these make more sense merged in with vehicle_status_flags.

eg
 - condition_data_link_valid
 - condition_avoidance_system_valid
 - condition_manual_control_valid

Reply:
Looks good to me. Can we assume this will be the final naming convention so that [#11457](https://github.com/PX4/Firmware/pull/11457) and [#11456](https://github.com/PX4/Firmware/pull/11456) can be adapted?
FYI @mrivi @MaEtUgR

Reply:
@cmic0 fine for me